app/code/Magento/ProductAlertGraphQl/README.md

```diff
@@ -0,0 +1,3 @@
+# ProductAlertGraphQl
```

Comment:
Please follow this template when adding a readme file: https://github.com/magento/devdocs/wiki/Magento-module-README.md